Assets/LeapMotion/Scripts/Animation/TransformTweenBehaviour.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Animation {
+
+  public class TransformTweenBehaviour : MonoBehaviour {
```

Comment:
We should either mark this with AddComponentMenu("") as an example script, or flesh out the interface a bit more.  Things like tween time and smoothing type for example.